### DISCUSSION TITLE
Numbered option keys in mod selection

### DISCUSSION THREAD

**Discussion by jvyden** [ppy/osu#18719] [Ideas] [answered]
Hello!

First of all, I love the new mod overlay. It looks very nice and it definitely looks like it's built to handle Lazer's massive amount of mods.

My one problem with it is my muscle memory has been sorta ruined. In stable, I can quickly clear my selected mods with this process:

1. F1 (Open mod select)
2. 1 (Deselect all mods)
3. 2 (Close)

This works because in Stable menu options can be pressed according to their number, shown in this screenshot
![image](https://user-images.githubusercontent.com/40577357/173892559-463f00d0-8c7f-4d4c-8992-b1ea145c6090.png)

I'm not sure why this worked in Lazer's old menu since mod selection isn't a menu, but that also leads to the question of why it was removed since it has to have been added manually.

I'll stop rambling now lol, this would be nice to have back is all.

**Comment by bdach** [comment, marked as answer]
The keybinds have been purposefully changed. You can use <kbd>Backspace</kbd> (rebindable in settings) to deselect all, and either <kbd>Escape</kbd> or <kbd>F1</kbd> to exit to song select.

**Reply by jvyden** [reply]
Oops. I guess I could've put in a little more effort looking for an option... >.<

**Reply by integerrr** [reply]
> ...and either <kbd>Escape</kbd> or <kbd>F1</kbd> to exit to song select.

<kbd>Enter</kbd> would also exit to song select
